exercises/es/exc_03_14_02.py

```diff
@@ -1,9 +1,9 @@
 import json
 import spacy
 
-nlp = spacy.load("en_core_web_sm")
+nlp = spacy.load("es_core_news_sm")
 
-with open("exercises/en/tweets.json") as f:
+with open("exercises/es/tweets.json", encoding="utf8") as f:
```

Comment:
Same question here, are the results okay or too confusing?

```
(Walmart, McDonalds, amo McDonalds) (Dante, Kid's Meal, McMenu, Quesoburguesa, McDonalds) (McDonalds,) (Gracias Mcdonalds,) (McDonalds,)
```

Reply:
Yes, these make sense (mostly)! I just have to modify the "amo McDonalds" part, since it included the verb as well.